repl/mock.go

```diff
@@ -1,6 +1,9 @@
 package repl
 
-import "github.com/spacemeshos/go-spacemesh/accounts"
+import (
+	"github.com/spacemeshos/CLIWallet/common"
+	// types "github.com/spacemeshos/go-spacemesh/common/types"
```

Comment:
undo this change?

Reply:
it is an old leftover. This whole thing needs a major update + more tests. Outside the scope of this pr.